pkg/v6/configmap/mock_test.go

```diff
@@ -43,6 +43,10 @@ func (h *helmMock) EnsureTillerInstalled(ctx context.Context) error {
 	return nil
 }
 
+func (h *helmMock) EnsureTillerInstalledWithValues(ctx context.Context, values []string) error {
+	return nil
+}
```

Comment:
Yes this is a pain. I only integrated `helmclienttest` into the latest version bundles. Once the KVM upgrades are done we can delete a lot of code.

We're carrying a lot of version bundles at the moment. This is because we can only delete pkg code when it's not used for all providers.